### ISSUE TITLE
new dataset: top 2015 south (blue and purple area)

### ISSUE THREAD

**Issue by mitchellryant** [TNRIS/tnris.org#945]
- [x] Add data to measures spreadsheet @gmullins
- [x] Route Data Shipment [copy data to TNRIS Projects share] @gmullins
- [x] Data Processing [renaming file, data organization] @mitchellryant
- [x] Generate MX DOQQs as JPEG2000s @mitchellryant 
- [x] Spotcheck MX JPEG2000s @gmullins
- [x] Aquisition dates on MX DOQQ filenames @mitchellryant 
- [x] Metadata for MX DOQQs @gmullins
- [x] Name and make shp of Missing DOQQs @mitchellryant 
- [x] Check Missing DOQQs @gmullins
- [x] Quality Assurance South @gmullins
- [x] Quality Assurance MX @gmullins
- [x] Download Prep [zip, copy to s3 bucket, coverage map] @mitchellryant
- [x] Move data to production shares @mitchellryant
- [x] Add to Data Download @wilsaj https://github.com/TNRIS/data-download/issues/77
- [x] Full State Inspection Sheet @gmullins
- [x] ONLY SUP REPORTS NEEDED: Data Publication [Update data catalog sheet, create data catalog images, supplemental reports staging] @gmullins
- [x] NOT NEEDED: Data Story @gmullins
- [x] Final Publication @christoball @gmullins
- [x] Inform TNRIS staff of data availability @gmullins
- [x] Add supplemental reports to website and we are done and can close this Issue - yay! @christoball 

![tx_remaining_deliveries](https://cloud.githubusercontent.com/assets/4921113/14506774/17e07e70-0185-11e6-9159-7ff1f04e961d.jpg)

**Comment by christoball** [comment]
Is this the final addition? 

Either way, we should start prepping the final announcement to make a big splash @gmullins

**Comment by gmullins** [comment]
Agreed @christoball  I was thinking the exact same thing. Prob a week or two out from all data available with final checks, etc.

**Comment by gmullins** [comment]
@gmullins Request MX tile metadata from Surdex.

**Comment by mitchellryant** [comment]
![image](https://cloud.githubusercontent.com/assets/8459928/14647423/51447e44-0623-11e6-82c1-4322f27348e4.png)

@gmullins for the mexico data we have DOQQs that do not intersect seamlines.

**Comment by gmullins** [comment]
@mitchellryant Ugh......and they listed a range in the metadata.

**Comment by mitchellryant** [comment]
@gmullins any file without a flight was not in new index and might be a completely black tile.

![image](https://cloud.githubusercontent.com/assets/8459928/14712699/cb61293a-07a3-11e6-98f0-2f20b71b7c2f.png)

![image](https://cloud.githubusercontent.com/assets/8459928/14712740/f82c4ff8-07a3-11e6-91ec-cb04fa3115e0.png)

**Comment by gmullins** [comment]
@mitchellryant top_mx_nodata_tiles.txt has the ones to be deleted. I'm pretty sure it's all the ones with no date. The rest of imagery that is already covered by other tiles - v.v.small areas no one will miss.

**Comment by gmullins** [comment]
296 DOQQs in wrong utm zone. waiting on replacement tifs, jp2s and metadata from Surdex.

**Comment by mitchellryant** [comment]
all 296 doqq are projected correctly and have been added to s3 and production location. 

consolidation of tifs continues.

**Comment by gmullins** [comment]
27 Tif 'Invalid Raster' DOQQs requested from Surdex.

**Comment by gmullins** [comment]
@mitchellryant @christoball Supplemental reports ready: texas_orthoimagery_program_top_2015_50cm_nc_cir_supplementalreports.zip

**Comment by mitchellryant** [comment]
"T" tag has been added to master data catalog sheet. will show on next update of sheet. or if @christoball manually adds T for that record in csv.
